src/lib/parser.ts

```diff
@@ -111,7 +111,7 @@ export function getLastSectionName(gptString: string): string {
 
 export function parseFunctionCall(text: string): FunctionCall {
   const functionCallRegex = /(\w+)\(([^)]*)\)/;
-  const argumentRegex = /([\w-]+)=({.*?}|'.*?'|".*?"|\[.*?\]|[^,]*)/g;
+  const argumentRegex = /([^,\s]+?)=({.*?}|'.*?'|".*?"|\[.*?\]|[^,]*)/g;
```

Comment:
The problem, which I've also added a test for, is some arguments contain dots in them (e.g. `data.account.id`)